.gitignore

```diff
@@ -1,3 +1,5 @@
 /node_modules
 .tern-port
 /dist
+
+*.iml
```

Comment:
Please don't mix random tweaks related to your build environment into pull requests. See also the various things added to `.npmignore`.